Update bedpe aggregator to use chromsizes-file

pkerpedjiev opened the pull request:
## Description

Fixed the bedpe aggregator to work with chromsizes files. It used to work with them but at some point the signatures of `bedpe` and `_bedpe` diverged and it broke.

I also harmonized the column numbers across bedpe and _bedpe so that they're always 1-based.

## Checklist

- [x] Unit tests added or updated
- [x] Updated CHANGELOG.md

mccalluc commented on clodius/cli/aggregate.py:
```diff
@@ -263,7 +267,7 @@ def line_to_dict(line):
         entries = [line_to_dict(first_line)]
 
-    entries += [line_to_dict(line.strip()) for line in f]
+    entries += [line_to_dict(line.strip()) for line in f if len(line.strip())]
```
Since `line.strip()` is repeated, it might be more clear to pull it out? And I don't think the `len()` is needed?
```python
entries += [line_to_dict(line) for line in [line.strip() for line f] if line]
```

pkerpedjiev replied:
Good point.

mccalluc commented on test/bedpe_test.py:
```diff
@@ -48,3 +48,23 @@ def test_clodius_aggregate_bedpe():
+    input_file = op.join(testdir, 'sample_data', 'galGal6.bed')
+    chromsizes_file = op.join(testdir, 'sample_data', 'galGal6.chrom.sizes')
+
+    output_file = '/tmp/blah.bed2ddb'
```
It's a little bit tidier to use the tempfile utils:
```
with tempfile.TemporaryDirectory() as tmpdirname:
...
```
- The outputs could potentially be large, and it's nice to clean up.
- If we get around to testing the output 😄, it would be better to be sure we're starting with a clean slate; If we use the same path every time, we might just be seeing the results of a previous run.

pkerpedjiev replied:
Done!

mccalluc commented:
```
ERROR: Use galGal6 chromsizes file
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/miniconda/lib/python3.6/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/home/travis/build/higlass/clodius/test/bedpe_test.py", line 69, in test_clodius_aggregate_bedpe2
    has_header=True)
  File "/home/travis/build/higlass/clodius/clodius/cli/aggregate.py", line 181, in _bedpe
    f = open(filepath, 'r')
FileNotFoundError: [Errno 2] No such file or directory: '/home/travis/build/higlass/clodius/test/sample_data/galGal6.bed'
-------------------- >> begin captured stdout << ---------------------
output_file: /tmp/blah.bed2ddb
plain
```

That file doesn't exist: Maybe use one that's checked in already?
```
clodius$ ls -s test/sample_data/*.bed
112 test/sample_data/cnv.bed
  8 test/sample_data/exon_unions_mm10.bed
  8 test/sample_data/exons.bed
  8 test/sample_data/geneAnnotationsExonsUnions.hg19.short.bed
 72 test/sample_data/geneAnnotationsExonsUnions.short.bed
144 test/sample_data/geneAnnotationsExonsUnions.short.multires.bed
 72 test/sample_data/geneAnnotationsExonsUnions1.short.bed
 72 test/sample_data/geneAnnotationsExonsUnions2.short.bed
784 test/sample_data/isidro.bed
  8 test/sample_data/sample_gwas.bed
  8 test/sample_data/test_float_importance.bed
```

pkerpedjiev commented:
> It would be nice to confirm the output... Would it be hard to check in the expected output as fixture, and then if anything changes, you'd notice?

Good idea! Just added it. And I added the missing file from the other comment.
